pkg/stub/handler.go

```diff
@@ -24,27 +25,26 @@ const (
 	clusterConfigResource = "cluster-config-v1"
 )
 
-func NewHandler() *Handler {
+func NewHandler(namespace string, manifestFactory *manifests.Factory) *Handler {
 	return &Handler{
-		manifestFactory: manifests.NewFactory(),
+		namespace:       namespace,
+		manifestFactory: manifestFactory,
 	}
 }
 
 type Handler struct {
+	namespace       string
 	manifestFactory *manifests.Factory
 }
 
 func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
```

Comment:
What does ctx hold with this new scheme (not useful at this time)?

Reply:
Need to look at operator-sdk code, not sure. Nothing important for now AFAIK.